apps/google-analytics-4/lambda/config/serverless-env.test.yml

```diff
@@ -1 +1,2 @@
-SIGNING_SECRET: ${env:SIGNING_SECRET}
+SIGNING_SECRET: ${ssm:/aws/reference/secretsmanager/ci/apps/google-analytics-4-action/test/signing-secret}
```

Comment:
See my other comment, for conistency let's do `signedSecret` here and in the other templates.